src/services/BlobStore.jl

```diff
@@ -152,6 +191,74 @@ end
 
 ##
 
+
+function addBlob!(
+    store::NavAbilityBlobStore,
+    filepath::AbstractString,
+    blobId::UUID = uuid4();
+    chunkSize::Integer = UPLOAD_CHUNK_SIZE_HASH,
+)
```

Comment:
We should rather call this `uploadFile` and have an `addBlob!` that matches the api. 
I haven't had a use case for `uploadFile` yet, but it's good to have as I can see some uses.